### ISSUE TITLE
only shows common name in subject

### ISSUE THREAD

**Issue by danie-dejager** [chris2511/xca#419]
Can the Subject view also show the SAN?

![image](https://user-images.githubusercontent.com/13314239/223955747-3137d555-5f22-43a8-9d21-73c266fda6c3.png)

**Comment by chris2511** [comment]
The SAN is not part of the subject, but part of the Extensions.
You probably knew this and wanted to suggest a UI improvement?

**Comment by daniejstriata** [comment]
Yes Please. It will make my life a little easier if the UI can show both.

**Comment by daniejstriata** [comment]
I was rather thinking that only showing the CommonName under the subject might be technically correct, it is not the only value of importance. Showing the SAN here would've made sense to me as I have a couple of certs with all pertinent info in the SAN and the CommonName being of little interest to me. From here I could take a screenshot to share with my users to show the domains or IP entries for the cert.

Adding the extra column does help though.
